content/python/concepts/regex/regex.md

```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Regular Expressions'
```

Comment:
Are the methods that are past of this entry supposed to be titled `re.methodname()` instead of `.methodname()`?